librz/analysis/rzil/rzil_trace.c

```diff
@@ -0,0 +1,216 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_analysis.h>
+
+/**
+ * Rzil trace should also these info
```

Comment:
`Rzil` -> `RZIL`

Reply:
Please use `RZIL` everywhere in the comments, except function or parameter names, and file names.